osu.Game.Rulesets.Mania/UI/Column.cs

```diff
@@ -38,6 +44,7 @@ public class Column : ScrollingPlayfield, IKeyBindingHandler<ManiaAction>, IHasA
         internal readonly Container TopLevelContainer;
         private readonly DrawablePool<PoolableHitExplosion> hitExplosionPool;
         private readonly OrderedHitPolicy hitPolicy;
+        private readonly SkinnableSound[] hitSounds;
```

Comment:
Is there a specific reason to store these to their own array rather than keep a reference to a `Container<SkinnableSound>` and access its children directly? I think that may read a touch better.